addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
+                    isHoliday = false;
```

Comment:
Yes, in the linked code the OnOffType is used also, and it is imported from "org.eclipse.smarthome.core.types.Type". I did find and use the same, however I was wondering why the IDE made ONLY suggestions to import from "org.openhab.core..", while Kai made suggestions not to import those. 
Should I hammer that into my aging brain or is there a setting for the IDE which I missed?